### ISSUE TITLE
contentWidth

### ISSUE THREAD

**Issue by jhihn** [qmlweb/qmlweb#262]
Can this property (contentWidth) be supported?

http://doc.qt.io/qt-5/qml-qtquick-text.html

**Comment by ChALkeR** [comment]
Yes, I suppose that's possible, but no time estimation for now — there are more important things that need to be done first.

**Comment by jhihn** [comment]
What are they? (just so I know what's going on)

**Comment by stephenmdangelo** [comment]
I have a commit that adds paintedWidth/Height (the old QtQuick 1 version of contentWidth/Height). May be useful: https://github.com/stephenmdangelo/qmlweb/commit/3742e60864dbba5ff5d1f9c2492439c80bce9aa3

It just returns the implicitWidth/Height, so is only good for non-wrapping text, I believe. But at least it allows any existing QML code that reads those properties.

**Comment by jhihn** [comment]
Fantastic news! Thank you.
